tests/integration/test_fim/test_synchronization/test_response_timeout.py

```diff
@@ -124,7 +124,7 @@ def wait_agent_dbsync_finish():
     def wait_agent_integrity_control():
         previous_time = datetime.now()
         while datetime.now() - previous_time < timedelta(seconds=1):
-            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command("sudo cat /var/ossec/logs/ossec.log")
+            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command("sudo cat /var/ossec/logs/wazuh.log")
```

Comment:
Here too.

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/20fb78d2a4218a84aa5ce2a4323ff3437686b23b